release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1669,6 +1669,147 @@ If these values are not defined in a failure domain configuration, for instance
+
+* Previously, when a worker node was trying to join a cluster the rendezvous node rebooted before the process completed. As the worker node could not communicate as expected with the rendezvous node, the installation was not successful. With this release, a patch is applied that fixes the racing condition that caused the rendezvous node to reboot prematurely and the issue is resolved.
+(link:https://issues.redhat.com/browse/OCPBUGS-41811[*OCPBUGS-41811*])
+
+* Previously, when using the {ai-full}, selecting a multi-architecture image for `s390x` CPU architecture on {hybrid-console} could cause the installation to fail. The installation program reported an error that the new cluster was not created because the skip MCO reboot was not compatible with `s390x` CPU architecture. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41716[*OCPBUGS-41716*])
+
+* Previously, when you selected a shared Virtual Private Cloud (VPC) during installation the bindings for the control plane service account remained. With this release, a fix ensures these bindings are deleted and no artifacts remain. (link:https://issues.redhat.com/browse/OCPBUGS-41538[*OCPBUGS-41538*])
+
+* Previously, a coding issue caused the Ansible script on {rh-openstack} user-provisioned installation infrastructure to fail during the provisioning of compact clusters. This occurred when IPv6 was enabled for a three-node cluster. With this release, the issue is resolved and you can provision compact three-node clusters
+on {rh-openstack} for user-provisioned installation infrastructure. (link:https://issues.redhat.com/browse/OCPBUGS-39402[*OCPBUGS-39402*])
+
+* Previously, the order of an Ansible Playbook was modified to run before the `metadata.json` file was created, which caused issues with older versions of Ansible. With this release, the playbook is more tolerant of missing files to accommodate older versions of Ansible and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-39285[*OCPBUGS-39285*])
+
+* Previously, when you installed a cluster there were issues using a worker node because the date, time, or both might have been inaccurate. With this release, a patch is applied to the live ISO time synchronization. The patch configures the `/etc/chrony.conf` file with the list of the additional Network Time Protocol (NTP) servers that the user provides in the `agent-config.yaml` file, so that you can use a worker node without experiencing a cluster installation issue.
```

Comment:
```suggestion
* Previously, when you installed a cluster there were issues using a compute node because the date, time, or both might have been inaccurate. With this release, a patch is applied to the live ISO time synchronization. The patch configures the `/etc/chrony.conf` file with the list of the additional Network Time Protocol (NTP) servers that the user provides in the `agent-config.yaml` file, so that you can use a worker node without experiencing a cluster installation issue.
```